integr8ly/integreatly-operator
# MGDAPI-5248 - bump: oauth proxy image

# Issue link

https://issues.redhat.com/browse/MGDAPI-5248

# What

Bump oauth proxy image for better image grading

# Verification steps



* Install RHOAM from this branch
```
make cluster/prepare/local
make code/run
```
* Once installed, verify oauth proxy image is updated
```
oc get deployment grafana-deployment -n local-rhoam-customer-monitoring-operator -o json | jq '.spec.template.spec.containers[1].image'
```
* Verify you can login into the customer grafana UI
